src/pb_stub.h

```diff
@@ -348,4 +351,70 @@ class Stub {
   std::mutex dlpack_proxy_stream_pool_mu_;
   std::unordered_map<int, cudaStream_t> dlpack_proxy_stream_pool_;
 };
+
+template <typename MessageType>
+void
+Stub::PrepareMessage(
+    AllocatedSharedMemory<MessageType>& msg_shm, MessageType** msg)
```

Comment:
I think you can remove `msg` as well from the argument list.

Reply:
I was wondering why we can remove `msg` from the argument list. This function is to initialize some variables of the `msg`, for example, the mutex and the condition variable.

Reply:
As far as I can tell from the code, you can access `msg` from `msg_shm` so I'm not sure why you need both of them to be passed.

Reply:
I understand it now. Updated the `PrepareMessage`. Thanks for the clarification!